examples/simple/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "tachijs-simple",
+  "version": "1.0.0",
+  "description": "Simple example for tachijs",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "ts-node src/"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/BoostIO/tachijs.git"
+  },
+  "keywords": [
+    "simple-example",
+    "tachijs"
+  ],
+  "author": "Sheenan Tenepre",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/BoostIO/tachijs/issues"
+  },
+  "homepage": "https://github.com/BoostIO/tachijs#readme",
+  "dependencies": {
+    "reflect-metadata": "^0.1.12",
+    "tachijs": "^0.4.1",
+    "tslib": "^1.9.3"
```

Comment:
Fix this too.

Reply:
@Rokt33r What do you mean? Does `ts-node src/` having errors on your end?

Reply:
Check https://github.com/BoostIO/tachijs/pull/11#discussion_r245664912